Translation into Brazilian Portuguese and corrections to documents in English

luizcmarin opened the pull request:
Note the following:
- created folders according to the standard for documentation
- all links checked

luizcmarin committed:
Translation into Brazilian Portuguese and corrections to documents in English

Note the following:
- created folders according to the standard for documentation
- all links checked

luizcmarin committed:
remove file lixo.md  (trash)

vjik commented on README.md:
```diff
@@ -85,12 +80,11 @@ To enable Xdebug, set `PHP_ENABLE_XDEBUG=1` in .env file
 so you have to configure your IDE to receive connections from that ip.
 
-If you are using macOS, you can fill `xdebug.remote_host` with `host.docker.internal`, due to a network limitation on mac (https://docs.docker.com/docker-for-mac/networking/#port-mapping)
+If you are using macOS, you can fill `xdebug.remote_host` with `host.docker.internal`, due to a network limitation on mac (<https://docs.docker.com/docker-for-mac/networking/#port-mapping>)
```
`<>` not need

luizcmarin replied:
I'm following Markdown rules. Do I make your change or follow the rules?
[markdown md034.md](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md034.md)

vjik replied:
GitHub automatically add links. But it's OK, you can keep angle brackets 👍

vjik commented on docs/guides/en/install-application-template.md:
````diff
@@ -1,18 +1,24 @@
+
+When running Apache you need a configuration like the following to use `enablePrettyUrl` in `.htaccess` in your public `public` folder:
+
+```html
````
```suggestion
```apache
```

vjik commented on docs/guides/en/install-application-template.md:
````diff
@@ -1,18 +1,24 @@
-    http://127.0.0.1:8000
-    
-When running Apache you need a configuration like the following to use `enablePrettyUrl` in `.htaccess` in your public `public` folder
+Open in your browser:
 
+```html
+     http://127.0.0.1:8000
+```
````
```suggestion
Open in your browser: `http://127.0.0.1:8000`
```

vjik commented on docs/guides/en/install-extensions.md:
```diff
@@ -1,52 +1,54 @@
+# Installing additional extensions
 
-It is recommended to install additional PHP extensions with https://github.com/mlocati/docker-php-extension-installer
+It is recommended to install additional PHP extensions with <https://github.com/mlocati/docker-php-extension-installer>
```
`<>` not need

luizcmarin replied:
I'm following Markdown rules. Do I make your change or follow the rules?
[](https://github.com/DavidAnson/markdownlint/blob/v0.33.0/doc/md034.md)

vjik commented on docs/guides/en/install-extensions.md:
````diff
@@ -1,52 +1,54 @@
 
 ### Composer asset plugin
 
+```bash
````
```suggestion
```Dockerfile
```

vjik commented on docs/guides/en/install-extensions.md:
````diff
@@ -1,52 +1,54 @@
 
 #### Debian
 
+```bash
````
```suggestion
```Dockerfile
```

vjik commented on docs/guides/en/install-extensions.md:
````diff
@@ -1,52 +1,54 @@
 
 *TBD*
 
+```bash
````
```suggestion
```Dockerfile
```

vjik commented on docs/guides/en/install-extensions.md:
````diff
@@ -1,52 +1,54 @@
-     
+#### Debian (PHP 7)
+
+```bash
````
```suggestion
```Dockerfile
```

vjik commented on docs/guides/en/install-extensions.md:
````diff
@@ -59,11 +61,13 @@ RUN apt-get update && \
 
 #### Debian
 
+```bash
````
```suggestion
```Dockerfile
```

vjik commented on tests/requirements/views/web/index.php:
```diff
@@ -2,76 +2,85 @@
 /* @var $summary array */
 /* @var $requirements array[] */
-?><!DOCTYPE html>
+?>
+<!DOCTYPE html>
```
```suggestion
?><!DOCTYPE html>
```
There should be nothing before `<!DOCTYPE html>`

vjik commented on tests/requirements/views/web/index.php:
This file is OK as is. Change only http to https in link "https://www.yiiframework.com/"

vjik commented on docs/guides/en/test-framework.md:
````diff
@@ -0,0 +1,47 @@
+git clone https://github.com/yiisoft/yii-core _host-volumes/yii-core
+```
+
+```information
+This repository has been archived by the owner on Sep 9, 2019. It is now read-only.
+
+Yii Framework 3.0 core was split into several packages and the package was deleted.
+
+Repository is kept so closed issues could be reviewed for historical reasons.
+```
````
```suggestion
> This repository has been archived by the owner on Sep 9, 2019. It is now read-only.
>
> Yii Framework 3.0 core was split into several packages and the package was deleted.
>
> Repository is kept so closed issues could be reviewed for historical reasons.
```

vjik commented on docs/guides/en/test-framework.md:
````diff
@@ -0,0 +1,47 @@
+
+Enter the container with
+
+```dockerfile
````
```suggestion
```bash
```

vjik commented on docs/guides/en/test-framework.md:
````diff
@@ -0,0 +1,47 @@
+
+## Using a specific PHP version
+
+```dockerfile
````
```suggestion
```bash
```

luizcmarin committed:
Update docs/guides/en/install-application-template.md

Co-authored-by: Sergei Predvoditelev <sergey.predvoditelev@gmail.com>

luizcmarin committed:
Update docs/guides/en/install-application-template.md

Co-authored-by: Sergei Predvoditelev <sergey.predvoditelev@gmail.com>

luizcmarin committed:
Update docs/guides/en/install-extensions.md

Co-authored-by: Sergei Predvoditelev <sergey.predvoditelev@gmail.com>

luizcmarin committed:
Update tests/requirements/views/web/index.php

Co-authored-by: Sergei Predvoditelev <sergey.predvoditelev@gmail.com>

luizcmarin committed:
Update docs/guides/en/test-framework.md

Co-authored-by: Sergei Predvoditelev <sergey.predvoditelev@gmail.com>

luizcmarin committed:
Update docs/guides/en/test-framework.md

Co-authored-by: Sergei Predvoditelev <sergey.predvoditelev@gmail.com>

luizcmarin committed:
Update docs/guides/en/test-framework.md

Co-authored-by: Sergei Predvoditelev <sergey.predvoditelev@gmail.com>

luizcmarin committed:
Translation into Brazilian Portuguese and corrections to docs in English

Note the following:
- created folders according to the standard for documentation
- all links checked

luizcmarin committed:
Merge branch 'master' of https://github.com/luizcmarin/yii-docker

luizcmarin committed:
remove path "docs/guides". Create path  "docs/guide"

samdark commented on docs/guide/en/test-framework.md:
````diff
@@ -0,0 +1,47 @@
+Get the source and place it into a host-volume folder for mounting it into the container.
+
+```bash
+git clone https://github.com/yiisoft/yii-core _host-volumes/yii-core
````
Why it's there? yii-core is no longer valid.

samdark replied:
Looks like it's the doc that was transferred from root dir w/o any change.

luizcmarin replied:
because it and the other files were transferred from the root to the 'en' folder, as now there is the 'pt-BR' folder too. And who knows, maybe one day someone will translate it into another language.
PLEASE see this file in full, as it seems to me that it only refers to "/yii-core" which, as you said, is no longer used. In that case, why keep it?

samdark commented on docs/internals.md:
```diff
@@ -0,0 +1,40 @@
+
+## Static analysis
+
+The code is statically analyzed with [Psalm](https://psalm.dev/). To run static analysis:
```
It is not true.

luizcmarin replied:
```
## Static analysis

The code is statically analyzed with [Psalm](https://psalm.dev/). To run static analysis:

```shell
./vendor/bin/psalm
```

Remove?

samdark replied:
Yes. See https://github.com/yiisoft/yii-docker/tree/master/.github/workflows. There are no workflows except building images.

samdark commented on docs/internals.md:
```diff
@@ -0,0 +1,40 @@
+
+## Code style
+
+Use [Rector](https://github.com/rectorphp/rector) to make codebase follow some specific rules or
```
It is not true.

schmunk42 reviewed (changes requested):
Some checks are failing

luizcmarin committed:
Various fixes

luizcmarin committed:
Docs folder standardization and other fixes

schmunk42 commented:
@yiisoft/reviewers *continuous-integration* says **Issues have been identified with 3 files** - but what are the issues?

samdark commented:
@schmunk42 code style issues :)

luizcmarin committed:
Update internals.md

samdark commented on docs/internals.md:
```diff
@@ -0,0 +1,38 @@
+# Internals
```
Not needed since there's nothing in this repository that is described in this file.

luizcmarin replied:
removed

luizcmarin committed:
Update file

luizcmarin committed:
Update requirements.php

luizcmarin committed:
Update file

luizcmarin committed:
Update file

samdark commented on README.md:
````diff
@@ -71,12 +86,15 @@ Adjust the versions in `.env` if you want to build a specific version.
 
-    docker-compose build
-
+```dockerfile
````
```suggestion
```shell
```

samdark commented on README.md:
````diff
@@ -71,12 +86,15 @@ Adjust the versions in `.env` if you want to build a specific version.
 ## Testing
 
-    docker-compose run --rm php php /tests/requirements.php
+```dockerfile
````
```suggestion
```shell
```

samdark committed:
Update README.md

samdark committed:
Update README.md
